htdocs/core/ajax/constantonoff.php

```diff
@@ -72,7 +72,8 @@
 		if ($action == 'set') {
 			dolibarr_set_const($db, $name, $value, 'chaine', 0, '', $entity);
 		} elseif ($action == 'del') {
-			dolibarr_del_const($db, $name, $entity);
+			// We should save the constant as 0 instead of deleting the constant.
```

Comment:
I keep setting the value to '0' here because:

- lots of external modules will use this script
- the core module stock.php has a default const STOCK_ALLOW_NEGATIVE_TRANSFER set on 1, and uses ajax/constantonoff.php for this const.

As explained [in the previous (closed) PR](https://github.com/Dolibarr/dolibarr/pull/23050/files#r1038301378)

Reply:
We should revert option
STOCK_ALLOW_NEGATIVE_TRANSFER
Into
STOCK_DISALLOW_NEGATIVE_TRANSFER
because we want
- no option in setup = option is default
- we should not set a default option during module activation (because default should be nothing set)